src/chain/chain_accumulator.rs

```diff
@@ -293,12 +294,24 @@ mod test {
         ))
     }
 
-    fn random_section_info_sig_payload(rng: &mut MainRng) -> EventSigPayload {
+    fn random_section_info_sig_payload(
+        rng: &mut MainRng,
+    ) -> (EventSigPayload, RealBlsEventSigPayload, RealBlsPublicKeySet) {
+        let bls_keys = generate_bls_threshold_secret_key(rng, 2);
+        let bls_secret_key_share = bls_keys.secret_key_share(0);
```

Comment:
key_share always indexed from 0 ? 
shall there be a comment at least explain why?

Reply:
I don't think it make sense to comment here, it is the secret_key_share api.
I can make it a named constant as well.